src/app.rb

```diff
@@ -0,0 +1,125 @@
+require './book'
+require './classroom'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def all_booklist
+    puts 'There is no book found in the store' if
+    @books.empty?
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def all_peoplelist
+    puts 'No people found!' if @people.empty?
+    @people.each { |person| puts "[#{person.class}], Age: #{person.name}, Name: #{person.age}, Id: #{person.id}" }
+  end
+
+  def create_person
+    print 'Missing student or teacher? create student (1) or a teacher (2)
+        [Input the number]: '
+    option = gets.chomp
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'invalid input'
+    end
+    puts 'Person created successfully'
+  end
+
+  def create_teacher
+    print 'teacher age: '
+    age = gets.chomp
+
+    print 'teacher specialization: '
+    specialization = gets.chomp
+
+    print 'teacher name: '
+    name = gets.chomp
+
+    teacher = Teacher.new(age, specialization, name, parent_permission: true)
+    puts 'New teacher was created succefully'
+    @people.push(teacher)
+  end
+
+  def create_student
+    print 'student age: '
+    age = gets.chomp
+
+    print 'student name: '
+    name = gets.chomp
+
+    print 'Parent permission [Yes or No]: '
+    parent_permission = gets.chomp.downcase
+
+    case parent_permission
+    when 'no'
+      student = Student.new(age, name, parent_permission: false)
+      @people.push(student)
+    when 'yes'
+      student = Student.new(age, name, parent_permission: true)
+      @people.push(student)
+    end
+    puts 'New student was created successfully'
+  end
```

Comment:
- [ ] Please make sure that when the user creates a student, the student is created and displayed correctly. Currently, all created students will have their `name` value be `Unkown`. Please fix this behavior.
![image](https://user-images.githubusercontent.com/62290545/195613095-93f96d1e-b6f8-4a54-bcae-45ed53c02d73.png)